llarp/net/sock_addr.cpp

```diff
@@ -206,8 +206,8 @@ namespace llarp
   bool
   SockAddr::operator<(const SockAddr& other) const
   {
-    return (m_addr.sin6_addr < other.m_addr.sin6_addr)
```

Comment:
why does this hunk need to be changed?

Reply:
Way more than just this need to be fixed as well, I am amending the PR.